src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -103,6 +106,18 @@ export class DocumentSelectionRequestProcessor {
   		throw new AppError(HttpCodesEnum.BAD_REQUEST, "Missing details in SESSION or PERSON IDENTITY tables");
   	}
 
+		//Initialise Yoti Service base on session client_id
+		const clientConfig = getClientConfig(this.environmentVariables.clientConfig(), f2fSessionInfo.clientId, this.logger);
```

Comment:
`getClientConfig` throws on error, should it be wrapped in a try/catch so that we can return a response to the user?

Reply:
We are handling this in next line where we throw "Unrecognised client in request" error if clientConfig is null. I have tested this and works as expected.